src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -257,18 +257,36 @@ public function setArguments(array $arguments)
         return $this;
     }
 
+    /**
+     * Sets the properties to define when creating the service.
+     *
```

Comment:
When we put the phpdoc, we never put a partial one. So please add the `@param` here.